clog/metrics_reporter.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from contextlib import contextmanager
+
+from yelp_meteorite import create_counter
+from yelp_meteorite import create_timer
+
+METRICS_SAMPLE_PREFIX = 'yelp_clog.sample.'
+METRICS_SAMPLE_RATE = 1000  # TODO consider configuring this elsewhere
```

Comment:
Here we can use staticconf as in other places in the code. 1000 could be the default. We can probably also have a config to enable/disable metrics altogether.
See https://github.com/Yelp/yelp_clog/blob/0f32c1f0eab118779f5fb1295e40e02d6278ec02/clog/config.py